docs/content/docs/dev/table/data_stream_api.md

```diff
@@ -574,6 +636,297 @@ env.execute()
 
 {{< top >}}
 
+Batch Runtime Mode
```

Comment:
This whole section seems kind of general.  Do you think it should be in a separate section and then linked to?

Reply:
yes, it should be a dedicated page. but this is future work. we need a batch mode page that is independent of API without any DataStream API or Table API example.